runtime/genesis_builder_test.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+	sc "github.com/LimeChain/goscale"
+	"github.com/LimeChain/gosemble/frame/grandpa"
+	"github.com/LimeChain/gosemble/primitives/types"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/signature"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_CreateDefaultConfig(t *testing.T) {
+	rt, _ := newTestRuntime(t)
+	wantGc := []byte("{\"system\":{},\"aura\":{\"authorities\":[]},\"grandpa\":{\"authorities\":[]},\"balances\":{\"balances\":[]},\"transactionPayment\":{\"multiplier\":\"1\"}}")
+
+	res, err := rt.Exec("GenesisBuilder_create_default_config", []byte{})
+	assert.NoError(t, err)
+
+	resDecoded, err := sc.DecodeSequence[sc.U8](bytes.NewBuffer(res))
+	assert.Equal(t, wantGc, sc.SequenceU8ToBytes(resDecoded))
+}
+
+func Test_BuildConfig(t *testing.T) {
+	rt, storage := newTestRuntime(t)
+
+	gc := []byte("{\"system\":{},\"aura\":{\"authorities\":[\"5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY\"]},\"grandpa\":{\"authorities\":[[\"5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY\",1]]},\"balances\":{\"balances\":[[\"5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY\",1000000000000000000]]},\"transactionPayment\":{\"multiplier\":\"2\"}}")
+
+	_, err := rt.Exec("GenesisBuilder_build_config", sc.BytesToSequenceU8(gc).Bytes())
```

Comment:
The function returns a result. If the result is `[0]`, it is ok, otherwise there is an unhandled error.